### PR TITLE
docs(release-highlights): creates q3 release highlights.

### LINKED ISSUE #3737: Create Q3 release highlights
**Is this a new page or a change to an existing one?**
Existing one
**Describe the enhancement**
Update the release highlights page for the Q3 release.

### PR DESCRIPTION
**Pull request by edonehoo**
Closes #3737

### CODE REVIEW

**Comment by nicolethoen** [comment]
I had expected that these release highlights would be just appended above the previous release's release highlights - a little like a blog feed? WDYT? Lots of people are still not migrated to PF5 yet.

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 1]
```diff
@@ -1,11 +1,39 @@
 ---
 id: Release highlights
-title: Release highlights
+title: Q3 release highlights
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:3]:
```suggestion
title: Release highlights
```

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+
+### Clickable labels
+
+We added a new "clickable" variant to the labels component, so that you can use a label that acts as a button.
+
+To enable this behavior, we've added a new `onClick` property to the `<Label>` component, which can contain an action fit for your use case. 
+
+Note that you cannot use `onClick` if the `href` or `isEditable` properties are already passed into `<Label>`, since those align with link labels and editable labels, respectively.
+
+<Divider></Divider>
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:35]:
```suggestion
```

**Review by evwilkin — APPROVED** [review]
LGTM 🎉

**Review by tlabaj — COMMENTED** [review]
Looks great Erin!

**Review by mcoker — APPROVED** [review]
🎃👍

**Review by jessiehuff — APPROVED** [review]
LGTM! 😄

**Pull request merged**